polaris/ocean/tasks/manufactured_solution/forward.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, component, name, resolution, subdir, init):
                          resolution=resolution, mesh=init, init=init,
                          package='polaris.ocean.tasks.manufactured_solution',
                          yaml_filename='forward.yaml',
-                         graph_filename='culled_graph.info',
+                         graph_filename=f'{init.path}/culled_graph.info',
```

Comment:
With my suggestion above, this would be:
```suggestion
                         graph_target=f'{init.path}/culled_graph.info',
```

Reply:
Similar changes may be required in other tasks.